readme.md

```diff
@@ -0,0 +1,30 @@
+# OlympusExtra
+
+OlympusExtra is a Hades DLC mod which adds new gods and boons to the Hades game. This includes unique effects, new god curses, duo boons, Thesus's wrath calls, and more. 
+
+## Locating your Hades game folder for modding
+- **Steam**: `Library` > Right click on Hades > `Manage` > `Browse local files` 
+- **Epic Games**: Go to `C:\Program Files\Epic Games\Hades`
+
+-----------------------------
+## Pre-Requisites 
+1. Download and unzip [modimporter](https://github.com/SGG-Modding/sgg-mod-modimporter/releases/latest/download/modimporter-windows.zip) (follow installation instructions on the Nexus Mod page). The downloaded `modimporter.exe` should be placed in the `Hades\Content` folder
+1. Download and unzip [Mod Utility](https://github.com/SGG-Modding/sgg-mod-modutil/releases/download/v2.3.1/ModUtil.zip). Place the unzipped contents in the `Hades\Content\Mods` folder 
+
+-----------------------------
+## Installation Instructions
+1. Make sure you fulfill the above pre-requisites!
```

Comment:
You could propose to make a full copy of the Hades file in case modding doesn't work. But that's not required since Modimporter creates a backup. Just a suggestion.